### PR TITLE
UI: JWT Auth Browser Popup warning. Resolves: #10753

### PR DESCRIPTION
**Pull request by aphorise**
Clarify warning and message as per community suggestion (#10753) to be more explicit about blocked pop-ups.

### CODE REVIEW

File: changelog/17009.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+UI: Added JWT authentication warning message about blocked pop-up windows and web browser settings.
```
Comment (zofskeez) [inline changelog/17009.txt:2]: Looking at other changelog entries it seems the descriptor before the colon is lowercase. Would you mind updating UI to lowercase while you're in there?

**Review by zofskeez — CHANGES_REQUESTED** [review]
Thanks for tackling this! Just a couple of issues with the changelog entry but the message update looks good!

File: changelog/17009.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
```
[the remark this answers is not in the document]
Comment (aphorise) [reply changelog/17009.txt]: Thanks @zofskeez  - corrected.

File: changelog/17009.txt (hunk 1 above)
Comment (aphorise) [reply changelog/17009.txt]: Corrected - thanks again @zofskeez

**Review by zofskeez — APPROVED** [review]

**Review by hashishaw — APPROVED** [review]

**Comment by aphorise** [comment]
hey @chelshaw do you know if this will make it out with the next 1.12 release - I believe it's not made it out with the 1.12.0 GA?

**Comment by hellobontempo** [comment]
@aphorise - It looks like this got lost in the shuffle, sorry about that! The circleci tests aren't running which is prohibiting this PR from merging

**Comment by aphorise** [comment]
Closing in favour of #18787

**Pull request closed without merging**